framework/src/ics/InitialConditionWarehouse.C

```diff
@@ -35,6 +36,7 @@ InitialConditionWarehouse::addObject(std::shared_ptr<InitialConditionBase> objec
 {
   // Check that when object is boundary restricted that the variable is nodal
   const MooseVariableFEBase & var = object->variable();
+  const std::tuple<std::string, int> ic_key = std::tuple(var.name(), object->_my_state);
```

Comment:
```suggestion
  const auto ic_key = std::tuple(var.name(), object->_my_state);
```

Reply:
hmm I see this conversation is marked as resolved, but the suggested change hasn't been applied. Is github broken?

Reply:
No I was marking them resolved because I made the change on my end. Just trying to keep track of the ones I have left.